src/components/tasks/TaskLevante.vue

```diff
@@ -108,7 +109,7 @@ async function startTask(selectedAdmin) {
 
     const gameParams = { ...appKit._taskInfo.variantParams };
 
-    const levanteTask = new levanteTaskLauncher(appKit, gameParams, userParams, 'jspsych-target');
+    const levanteTask = new levanteTaskLauncher(appKit, gameParams, userParams, logger);
```

Comment:
@asengupta3 Just wanna make sure this is correct? Were no longer passing in the HTML element classname to attach the jsPsych instance to?